src/Connector.SqlServer/Connector/SqlServerConnector.cs

```diff
@@ -644,5 +644,15 @@ protected override async Task<bool> CheckTableExists(ExecutionContext executionC
 
             return await CheckTableExists(config, tableName);
         }
+
+        private static async Task ExecuteCommandWithRetryAsync(Task command)
+        {
+            await ActionExtensions.ExecuteWithRetryAsync(async () => await command);
```

Comment:
Need to specify which exceptions is transient with `ex.IsTransient()` from `ExceptionExtensions` as an argument to `ExecuteWithRetry`

Reply:
Done